From b5e29bc46e25241dec5d5295a945c2617bc66aaf Mon Sep 17 00:00:00 2001
From: Radek Vykydal <rvykydal@redhat.com>
Date: Thu, 7 Mar 2024 12:21:03 +0100
Subject: [PATCH 1/2] template

---
 share/templates.d/99-generic/runtime-install.tmpl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/share/templates.d/99-generic/runtime-install.tmpl b/share/templates.d/99-generic/runtime-install.tmpl
index 5ab3ca10..dcab9c39 100644
--- a/share/templates.d/99-generic/runtime-install.tmpl
+++ b/share/templates.d/99-generic/runtime-install.tmpl
@@ -136,6 +136,7 @@ installpkg tigervnc-server-module
 installpkg net-tools
 installpkg bridge-utils
 installpkg nmap-ncat
+installpkg prefixdevname
 
 ## hardware utilities/libraries
 installpkg pciutils usbutils ipmitool
-- 
2.43.0

